libsignify/src/consts.rs

```diff
@@ -0,0 +1,40 @@
+//! Constants and type defintions from the `signify` design.
```

Comment:
```suggestion
//! Constants and type definitions from the `signify` design.
```

Reply:
Ouch, thanks. I flipped a spellchecker on and took a look through the project and didn't see any other typos.